### PR TITLE
Fixed logic for checking for free machines on Sauce Labs (closes #31)

### LINKED ISSUE #31: Sometimes tasks get stuck on 'queued' state while running on Sauce Labs
Tasks can get stuck on 'queued' state, while free machines are available at that moment. This can continue untill the TravisCI will close a job on timeout
Example: https://travis-ci.org/testcafe-build-bot/testcafe/jobs/128272495#L1233

### PR DESCRIPTION
**Pull request by georgiy-abbasov**
\cc @AlexanderMoskovkin, @helen-dikareva

### CODE REVIEW

File: src/saucelabs/runner.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ export default class SauceLabsRunner extends EventEmitter {
     async _checkForFreeMachines () {
         var freeMachinesCount = await this._getFreeMachineCount();
 
-        if (freeMachinesCount)
+        if (freeMachinesCount > 0)
```
Comment (AlexanderMoskovkin) [inline src/saucelabs/runner.js]: It's not clear without the problem context why we can't use just `if (freeMachinesCount)`, please add the comment.

Comment (helen-dikareva) [inline src/saucelabs/runner.js]: I think method `_getFreeMachineCount` schould return `Math.max(0, ...)` to code will be more clearly

Comment (AlexanderMoskovkin) [inline src/saucelabs/runner.js]: yep. It'll be better. In this case we need not the comment

Comment (AlexanderMoskovkin) [inline src/saucelabs/runner.js]: and we should not change this condition

Comment (georgiy-abbasov) [inline src/saucelabs/runner.js]: Thanks, got it.

**Comment by georgiy-abbasov** [comment]
FPR

**Comment by helen-dikareva** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Pull request merged**